stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/mentions/MentionsFragment.kt

```diff
@@ -0,0 +1,28 @@
+package io.getstream.chat.ui.sample.feature.mentions
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import io.getstream.chat.ui.sample.databinding.FragmentMentionsBinding
+
+class MentionsFragment : Fragment() {
+
+    private var _binding: FragmentMentionsBinding? = null
+    private val binding get() = _binding!!
```

Comment:
Do we really need to have both `_binding` and `binding` as both are private? For me, it's almost the same as having a single `lateinit var binding: FragmentMentionsBinding`.

Reply:
We need the separate nullable `_binding` because we have to set it to `null` in `onDestroyView` to prevent leaks. You can't `null` out a `lateinit` variable.

Reply:
With lateinit we won't be able to release binding in `onDestroyView()`:
```
    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }
```

Leaving as is for now as the solution with delegate is too complex for the sample app:
https://medium.com/@Zhuinden/simple-one-liner-viewbinding-in-fragments-and-activities-with-kotlin-961430c6c07c

Reply:
Replaced glide with coil, introduced common spacing dimensions, introduced `unauthorizedUser` in HomeFragmentViewModel, fixed typo in `setBangeNumber` method